src/Servers/HttpSys/src/AuthenticationSchemes.cs

```diff
@@ -6,14 +6,40 @@
 namespace Microsoft.AspNetCore.Server.HttpSys
 {
     // REVIEW: this appears to be very similar to System.Net.AuthenticationSchemes
+    /// <summary>
+    /// Specifies protocols for authentication.
+    /// </summary>
     [Flags]
     public enum AuthenticationSchemes
     {
+        /// <summary>
+        /// No authentication is allowed. A client requesting an <see cref="AuthenticationManager"/> object with this flag set will
+        /// always receive a 403 Forbidden status. Use this flag when a resource should never be served to a client.
```

Comment:
```suggestion
        /// No authentication is enabled. This should only be used when HttpSysOptions.Authentication.AllowAnonymous is enabled.
```
Plus the appropriate `see` ref.